examples/python/encryptor_config_with_zone.yaml

```diff
@@ -46,4 +46,15 @@ schemas:
       - id
       - email
     encrypted:
-      - column: email
\ No newline at end of file
+      - column: email
+
+  - table: test_example_with_zone
+    columns:
+      - id
+      - data
+      - raw_data
+    encrypted:
+      - column: data
+        data_type: bytes
+        # base64 bytes value
```

Comment:
mention decoded value of base64 string to understand what we should expect: "test-data"